imap_processing/lo/tests/test_science_direct_events.py

```diff
@@ -0,0 +1,290 @@
+import bitstring
+
+from imap_processing.lo.l0.science_direct_events import ScienceDirectEvents
+
+# TODO: These unit tests don't call the public function for the ScienceDirectEvents
+# class and are also only able to run if the contects of __init__ are commented out
+# because I currently don't have any compressed DE data. As a workaround, I am
+# directly setting the DATA attribute and calling and testing the private methods
+# as needed. These tests will be updated when data becomes available. When that happens
+# the need for the bitstring import will also go away.
```

Comment:
I actually think this is great! Don't remove these once you get real data. Mock data is fantastic for checking things are working as expected in a simple to comprehend way without needing to realize what binary values are in a file.

Reply:
Sounds good. I can definitely continue to directly set variables for testing, but once I get data I'll be able to properly initialize the object rather than comment out `__init__`. I'll update that comment to reflect that.